reports/_outliers.qmd

```diff
@@ -179,7 +179,7 @@ ggplot(training_data, aes(
   )
 ```
 
-```{r fig.height=6}
+```{r _outliers_median_price_outliers fig.height=6}
```

Comment:
**issue:** Need a comma after the name here.

```suggestion
```{r _outliers_median_price_outliers, fig.height=6}
```